src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -55,7 +55,7 @@ class CallActor(call: Call, locallyQualifiedInputs: CallInputs, backend: Backend
   when(CallNotStarted) {
     case Event(Start, _) =>
       val backendInputs = backend.adjustInputPaths(call, locallyQualifiedInputs)
-      call.instantiateCommandLine(backendInputs) match {
+      call.instantiateCommandLine(backendInputs, backend.engineFunctions(backend.setupCallEnvironment(call, workflowDescriptor))) match {
```

Comment:
I was picturing TaskExecutionContext as being the "other stuff" that combines with Backend to create the context-aware backend. Do you view it as actually being the context-aware backend instead?

Either way I still think that we should be creating the fully context-aware backend outside of CallActor and passing it in instead of determining it inside.